ci/test/01_base_install.sh

```diff
@@ -7,7 +7,9 @@
 export LC_ALL=C.UTF-8
 
 CI_EXEC_ROOT () { bash -c "$*"; }
+CI_EXEC()       { bash -c "$*"; }
```

Comment:
Are these (doing) the same thing? I see they are used in [the same context](https://github.com/bitcoin/bitcoin/pull/27028/commits/fabcf65e46f1b5b5914b9738a87375ad750c5c7f#diff-b088baf93122a28210fe5ebdb9d5c3100813aac183b8362095c2c2d05051d70dR49-R52), immediately after one another. Why the new func? Maybe a comment to explain the difference?